security/advancedtls/crl.go

```diff
@@ -61,8 +61,13 @@ type RevocationOptions struct {
 	// Directory format must match OpenSSL X509_LOOKUP_hash_dir(3).
 	// Deprecated: use CRLProvider instead.
 	RootDir string
+	// DenyUndetermined controls if certificate chains with RevocationUndetermined
+	// revocation status are allowed to complete.
+	DenyUndetermined bool
```

Comment:
I'd prefer to keep the current ['positive boolean' approach](https://testing.googleblog.com/2023/10/improve-readability-with-positive.html), potentially with 'default true' mechanism like 
`
type MyConfig struct {
    AllowChanges bool
}

func NewMyConfig() MyConfig {
    return MyConfig{
        AllowChanges: true,
    }
}`
Leaving it to you&Doug to decide.

Reply:
I don't like this because it requires adding a constructor layer for a relatively simple struct - in this case IMO the positive boolean approach isn't worth the complexity

@dfawley WDYT?

Reply:
Yeah there's unfortunately come conflict between having positive booleans and having default behavior = zero value.

Maybe `DenyUndetermined` is not too negative?  It states what you _will_ be doing, not what you will _not_ be doing.  Or we could go with something more verbose that removes the double-negative like `RequireDetermined[Revocation]Status`?

Reply:
BTW I'm fine either way.  Another option could also be to replace with an "enum" (dumb Go doesn't actually have them, but):

```go
type RevocationStatusRequirement int

const (
    DenyUndetermined RevocationStatusRequirement = 0
    AllowUndetermined
)
```

(Names are just for illustrative purposes; probably pick better names.)